Trappist/src/Promact.Trappist.Core/Controllers/TestController.cs

```diff
@@ -11,30 +10,38 @@ namespace Promact.Trappist.Core.Controllers
     public class TestsController : Controller
     {
         private readonly ITestsRepository _testRepository;
-        private readonly IStringConstants _stringConstant;
-  
-        public TestsController(ITestsRepository testRepository, IStringConstants stringConstant)
+        public TestsController(ITestsRepository testRepository)
         {
             _testRepository = testRepository;
-          _stringConstant = stringConstant;
-        }    
+        }
+        /// <summary>
+        /// this method is to verify unique name of a test
+        /// </summary>
+        /// <param name="testName">name of the test</param>
+        /// <returns>boolean</returns>
+        [HttpGet("{testName}")]
```

Comment:
Violates basic http method rules, you are calling /api/tests/{testname} to check whether test name is unique or not - that is wrong url. How would u create new api for getting actual test by id or name in future? Change accordingly